# Bat1K 21families: analyses, code, and contributors
**Scripts, code, and contact information of the core team who contributed to the analyses in the flagship paper of the Bat1K project phase 1 (21 families).**<br>
We did not develop new software for this study; thus, we provide example commands for analyses or provide links to other sources employed.

### Content:
1. [Genome assembly](https://github.com/Bat1K-21families/21families-analyses/edit/main/README.md#1-genome-assembly-for-new-genomes-released-in-this-paper)
2. [Annotations](https://github.com/Bat1K-21families/21families-analyses/edit/main/README.md#2-annotations)
3. [Alignements](https://github.com/Bat1K-21families/21families-analyses/edit/main/README.md#3-alignements)
4. [Phylogenomic inference](https://github.com/Bat1K-21families/21families-analyses/edit/main/README.md#4-phylogenomic-inference-molecular)
5. [Molecular dating with fossil calibrations](https://github.com/Bat1K-21families/21families-analyses/edit/main/README.md#5-molecular-dating-with-fossil-calibrations)
6. [Fossilized birth-death models (molecular + morphology)](https://github.com/Bat1K-21families/21families-analyses/edit/main/README.md#6-fossilized-birth-death-models-molecular--morphology)
7. [Reconstructing the ancestral bat karyotype ](https://github.com/Bat1K-21families/21families-analyses/edit/main/README.md#7-reconstructing-the-ancestral-bat-karyotype)
8. [Biogeography ](https://github.com/Bat1K-21families/21families-analyses/edit/main/README.md#8-biogeography)


## Analyses:

### 1. Genome Assembly (new genomes released in this paper)
Led by Evgeny Leushkin, Thomas Brown, and **add authors**
**[see code repository]()**

### 2. Genome Annotation
Led by Michael Hiller **[see code repository]()**
- Coding genes (TOGA)

Led by David Ray **[see code repository]()**
- Transposable Elements (annotations and analyses)

Led by Bastian Fromm [see code repository](https://github.com/bioinfoUGR/bat1k)
- miRNAs annotations

Led by Zixia Huang **[see code repository]()**
- miRNAs analyses

### 3. Alignments
Led by Ariadna Morales **[see code repository](https://github.com/ariadnamorales/Bat1K_21Fam_Phylogenomics)**
- Coding genes (TOGA)
- Reference-based multi-genome alignments (MultiZ)

Led by Evgeny Leushkin **[see code repository]()**
- Reference-free multi-genome alignments (Cactus)

### 4. Phylogenomic Inference (molecular)
Led by Ariadna Morales **[see code repository](https://github.com/ariadnamorales/Bat1K_21Fam_Phylogenomics)**
- Coalescent-based species tree (from TOGA-annotated coding genes)
- Whole-genome phylogenies
- Chromosome-by-chromosome phylogenies
- Sliding window phylogenies
- Neutral intergenic window analyses

Led by Graham Hughes **[see code repository]()**
- Concatenation-based coding gene phylogeny

Led by Yan Liang **[see code repository]()**
- Neutral sites phylogeny
- Mitochondrial phylogenies

### 5. Molecular Dating with Fossil Calibrations
Led by Graham Hughes **[see code repository]()**

### 6. Fossilized Birthâ€“Death Models (molecular + morphological)
Led by William Thomas **[see code repository]()**

### 7. Ancestral Bat Karyotype Reconstruction
Led by Denis Larkin **[see code repository]()**

### 8. Biogeography
Led by William Thomas **[see code repository]()**
